include/swift/SIL/SILInstruction.h

```diff
@@ -7771,23 +7771,25 @@ class GraphOperationInst final
   unsigned NumOperands;
   /// The attributes of the graph operation.
   MutableArrayRef<GraphOperationAttribute> Attributes;
+  /// Whether this instruction is to be executed out of graph (via eager op
+  /// dispatch). For an op with dynamic attribute(s), it must run out of graph.
+  bool RunOutOfGraph;
```

Comment:
cc @dan-zheng @lattner 
I see that you added a new configuration in `graph_op`, but you didn't implement the parser and printer for it. This is pretty weird in IR design, because the textual representation should reflect the *exact* semantics of the in-memory format. The principled way is to add a `[no_clustering]` attribute to `graph_op`.

It's fine to not include the parser/printer code in this patch. Please file a bug and add a FIXME for this. Thanks!